### PR TITLE
media-libs/opencolorio: Update to v20170719

### PR DESCRIPTION
**Pull request by dracwyrm**
Upstream doesn't appear to be making new releases, so use stable
snapshots.

There was another round of yaml fixes and other fixes for security.

Build fixes for GCC 7.

General bug fixes.

The current version in portage has lots of bugs that have since
been fixed.

Signed-off by: Jonathan Scruggs <j.scruggs@gmail.com>

@SoapGentoo

### CODE REVIEW

File: media-libs/opencolorio/files/opencolorio-1.0.9-cmake-fixes-v1.patch [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+-###############################################################################
+ ### Create Doc Targets ###
+ 
+ message(STATUS "Create sphinx conf.py from conf.py.in")
+@@ -99,7 +36,7 @@ else()
+ endif()
+ 
+ add_custom_target(doc ALL
+-    COMMAND PYTHONPATH=${PYTHONPATH} ${EXTDIST_BINPATH}/sphinx-build -b html . ${CMAKE_CURRENT_BINARY_DIR}/build-html
++    COMMAND sphinx-build -b html . ${CMAKE_CURRENT_BINARY_DIR}/build-html
```
Comment (SoapGentoo) [inline media-libs/opencolorio/files/opencolorio-1.0.9-cmake-fixes-v1.patch]: 👍

File: media-libs/opencolorio/opencolorio-1.0.9_p20170719.ebuild [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE="doc opengl pdf python cpu_flags_x86_sse2 test"
+
+RDEPEND="opengl? (
+		media-libs/lcms:2
+		media-libs/openimageio
+		media-libs/glew:=
+		media-libs/freeglut
+		virtual/opengl
+		)
```
Comment (SoapGentoo) [inline media-libs/opencolorio/opencolorio-1.0.9_p20170719.ebuild]:
```
RDEPEND="
	opengl? (
		media-libs/lcms:2
		media-libs/openimageio
		media-libs/glew:=
		media-libs/freeglut
		virtual/opengl
	)
```

File: media-libs/opencolorio/opencolorio-1.0.9_p20170719.ebuild [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? (
+		pdf? ( dev-python/sphinx[latex,${PYTHON_USEDEP}] )
+		!pdf? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	)"
+
+# Documentation building requires Python bindings building
+REQUIRED_USE="doc? ( python )"
```
Comment (SoapGentoo) [inline media-libs/opencolorio/opencolorio-1.0.9_p20170719.ebuild]: `REQUIRED_USE="doc? ( python ) python? ( ${PYTHON_REQUIRED_USE} )"` (also move it directly below `IUSE`)

File: media-libs/opencolorio/files/opencolorio-1.0.9-cmake-fixes-v1.patch (hunk 1 above)
Comment (dracwyrm) [reply media-libs/opencolorio/files/opencolorio-1.0.9-cmake-fixes-v1.patch]: I though you would that this was finally fixed. :P

**Pull request closed without merging**